worker/pkg/workflows/datasync/activities/sync-redis-clean-up/activity.go

```diff
@@ -50,14 +61,12 @@ func DeleteRedisHash(
 		"RedisHashKey", req.HashKey,
 	)
 
-	// todo: this should be factored out of here and live on the activity itself
-	redisClient, err := neosync_redis.GetRedisClient()
-	if err != nil {
-		return nil, err
+	if a.redisclient == nil {
```

Comment:
Okay so I'm guessing the way this works is that you are providing an empty redis client but the activity is never invoked via the workflow unless we detect redis clients, yes?